substrafl_examples/strategies_examples/plot_substrafl_fedavg.py

```diff
@@ -403,9 +403,9 @@ def forward(self, x, eval=False):
 
 
 class TorchDataset(torch.utils.data.Dataset):
-    def __init__(self, x, y, is_inference: bool):
-        self.x = x
-        self.y = y
+    def __init__(self, datasamples, is_inference: bool):
+        self.x = torch.FloatTensor(datasamples["images"][:, None, ...])
+        self.y = F.one_hot(torch.from_numpy(datasamples["labels"]).type(torch.int64), 10).type(torch.float32)
```

Comment:
Why do we have to convert twice here?

Reply:
I guess, int for one hot encoding then float for the network (I took the code from the example so I can't say much)